examples/routing_service/routing_service_monitoring/c++11/USER_QOS_PROFILES.xml

```diff
@@ -0,0 +1,295 @@
+<?xml version="1.0"?>
+<!-- 
+Description
+XML QoS Profile for monitoring
+
+(c) Copyright, Real-Time Innovations, 2012.  All rights reserved.
+RTI grants Licensee a license to use, modify, compile, and create derivative
+works of the software solely for use with RTI Connext DDS. Licensee may
+redistribute copies of the software provided that all such copies are
+subject to this license. The software is provided "as is", with no warranty
+of any type, including any warranty for fitness for any purpose. RTI is
+under no obligation to maintain or support the software. RTI shall not be
+liable for any incidental or consequential damages arising out of the use
+or inability to use the software.
+
+The QoS configuration of the DDS entities in the generated example is loaded 
```

Comment:
You can get rid of all the boiler plate comments and leave just the usual:
```xml
<!-- 
 (c) 2021 Copyright, Real-Time Innovations, Inc.  All rights reserved.
 RTI grants Licensee a license to use, modify, compile, and create derivative
 works of the Software.  Licensee has the right to distribute object form only
 for use with RTI products.  The Software is provided "as is", with no warranty
 of any type, including any warranty for fitness for any purpose. RTI is under
 no obligation to maintain or support the Software.  RTI shall not be liable for
 any incidental or consequential damages arising out of the use or inability to
 use the software.
```

Reply:
Done!